abuild.in

```
@@ -61,6 +61,28 @@ error() {
 	logcmd "ERROR: $pkgname: $1"
 }
 
+amove() {
+	[ -n "${1##/*}"   ] || return 0
+	[ -n "$subpkgdir" ] || return 0
+	[ -e "$pkgdir/$1" ] || return 0
```

I think we should return error instead of silently do nothing.

---

Done, all pre-condition-based failures return 1.